tmpl/consent-banner.php

```diff
@@ -9,10 +12,17 @@
 ?>
 
 <div id="cookie-consent-banner">
-	<?php load_template( __DIR__ . '/consent-updated.php' ); ?>
+	<?php
+	/**
+	 * Allow the consent updated template path to be overridden so it can be customized independently. This template displays the "preferences updated" messaging after they have been saved.
+	 *
+	 * @var string The path to the consent updated template.
+	 */
+	apply_filters( 'altis.consent.consent_updated_template_path', load_template( __DIR__ . '/consent-updated.php' ) );
```

Comment:
Same thing, the default template will be always loaded. Or am I missing something?